tests/createGlobalState.test.ts

```diff
@@ -6,16 +6,68 @@ describe('useGlobalState', () => {
     expect(createGlobalState).toBeDefined();
   });
 
-  it('both components should be updated', () => {
+  it('both components should be updated directly', () => {
     const useGlobalValue = createGlobalState(0);
     const { result: result1 } = renderHook(() => useGlobalValue());
     const { result: result2 } = renderHook(() => useGlobalValue());
-    expect(result1.current[0] === 0);
-    expect(result2.current[0] === 0);
+    let [ state1, setState1 ] = result1.current;
+    let [ state2 ] = result2.current;
+    expect(state1 === 0);
```

Comment:
I don't think this does anything. You need to follow `expect` with `to...`. So this should be `expect(state1).toBe(0)`.

Reply:
Fixed